apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenValue.vue

```diff
@@ -2,16 +2,7 @@
   <div class="d-code--sm d-fc-purple-400 d-wmx164">
     <div v-if="isCompositionToken(tokenValue)">
       <span v-for="value in tokenValue" :key="value">
-        <span v-if="valueIsDivided(value)">
-          <span v-dt-tooltip="getTooltipValue(getCompositionTokenNumerator(value))" class="h:d-fc-secondary">
-            {{ getCompositionTokenNumerator(value) }}
-          </span>
-          /
-          <span v-dt-tooltip="getTooltipValue(getCompositionTokenDenominator(value))" class="h:d-fc-secondary">
-            {{ getCompositionTokenDenominator(value) }}&nbsp;
-          </span>
-        </span>
-        <span v-else v-dt-tooltip="getTooltipValue(value)" class="h:d-fc-secondary">
+        <span v-dt-tooltip="getTooltipValue(value)" :class="{ 'h:d-fc-secondary': getTooltipValue(value) }">
```

Comment:
Why add this condition to `h:d-fc-secondary`? Is there a case where `getTooltipValue(value)` is undefined?

Reply:
Yeah, it's only for the /